### DISCUSSION TITLE
Path to use to locate custom bash script in custom Homebrew cask?

### DISCUSSION THREAD

**Discussion by lancejpollard** [Homebrew/discussions#5079] [Casks]
### Output of `brew config`

```shell
N/A
```

### Output of `brew doctor`

```shell
N/A
```

### Description of issue

Crossposting for visibility https://stackoverflow.com/questions/77863275/path-to-use-to-locate-custom-bash-script-in-custom-homebrew-cask

I have successfully installed a custom Homebrew cask using https://github.com/termsurf/homebrew-load:

    $ brew install --cask termsurf/load/task

However, it fails at the [last step](https://github.com/termsurf/homebrew-load/blob/059d1b7e329fd466374eed6f2aa57e2ec01960b7/Casks/task.rb#L36):

    system_command "install.sh"

I tried as well:

    system_command "#{staged_path}/install.sh"

But both fail with corresponding errors:

    Error: termsurf/load/task: Failure while executing; `/usr/bin/env install.sh` exited with 127. Here's the output:
    env: install.sh: No such file or directory

What is the path to use to get access to that custom install script located at the base of my repo (or where should I put it in the repo)?

Also, what is the [`installer` DSL](https://github.com/Homebrew/homebrew-cask/blob/006622a075baf969440af1a64ffa7da5dc6bf422/Casks/j/j.rb#L20)? Is that relevant here?

Thanks for the help!

### Relevant casks

https://github.com/termsurf/homebrew-load/blob/059d1b7e329fd466374eed6f2aa57e2ec01960b7/Casks/task.rb

**Comment by SMillerDev** [comment]
> What is the path to use to get access to that custom install script located at the base of my repo (or where should I put it in the repo)?

No such path is supported. Casks are supposed to distribute external software.

**Reply by lancejpollard** [reply]
So what are my options?

**Reply by SMillerDev** [reply]
Nothing really, that's not a supported situation for casks.
